### ISSUE TITLE
[Feature Request]  A way to fine-tune the stereo spread of the delay.

### ISSUE THREAD

**Issue by bent95** [TheWaveWarden/odin2#347]
Currently there is only one button controlling the stereo width/spread of the delay: Ping-Pong off, meaning no changes to the pan; and Ping-Pong on, meaning the delay continually pans from 100% to the left to 100% right, giving a very prominent stereo effect. This is pretty limiting, because often you’d want to use something in-between the two extremes…

This issue could easily be solved by adding a knob adjusting the spread of the ping-pong effect, where having a value of 0.5 would make the delay pan between 50% to the left and 50% right, while the value 1.0 would behave identical to the current Ping-Pong on setting.

**Comment by TheWaveWarden** [comment]
Yes this would be nice. I have to see how this works out from a GUI-layout perspective.

@bent95 You can also control this in your DAW by limiting the stereo spread of the according mixer channel.
